client/request.go

```diff
@@ -134,8 +134,12 @@ func (cli *Client) doRequest(ctx context.Context, req *http.Request) (serverResp
 
 		// Don't decorate context sentinel errors; users may be comparing to
 		// them directly.
-		switch err {
-		case context.Canceled, context.DeadlineExceeded:
+		if err == context.Canceled {
```

Comment:
Still needs to detect `context.DeadlineExceeded` to match previous behavior

Reply:
Nevermind per comments on another mention of the change. I didn't realize this implemented timeoutError.